packages/rancher-monitoring/generated-changes/overlay/files/rancher/home/rancher-default-home.json

```diff
@@ -1238,7 +1238,7 @@
       "list": []
     },
     "time": {
-      "from": "now-6h",
+      "from": "now-5m",
```

Comment:
What does this do since it's significantly shorter?

Reply:
Instead of showing metrics for the last 6h on Grafana by default, it shows 5m by default. You can always increase the scope on Grafana though.

e.g. if you see the graphs here, the x axis is scoped to the last 5 minutes.

![Screen Shot 2021-04-07 at 1 14 56 PM](https://user-images.githubusercontent.com/13127443/113944954-4149d100-97ba-11eb-87f3-3ac16c6b3e27.png)